extracts/v_extracts.whetstone_users.sql

```diff
@@ -128,15 +128,18 @@ FROM
            ,scw.primary_on_site_department AS course_name
            ,scw.preferred_first_name + ' ' + scw.preferred_last_name AS [user_name]
            ,CASE WHEN scw.primary_site_schoolid <> 0 THEN scw.primary_site END AS school_name
-           ,CAST(CASE WHEN scw.[status] = 'TERMINATED' THEN 1 ELSE 0 END AS BIT) AS inactive
+           ,CAST(CASE WHEN scw.[status] = 'TERMINATED' THEN 1 
+                      WHEN scw.primary_job = 'Paraprofessional' THEN 1
+                      ELSE 0 END AS BIT) AS inactive
            ,CASE WHEN scw.grades_taught = 0 THEN 'K' ELSE CAST(scw.grades_taught AS VARCHAR) END AS grade_abbreviation
            ,CASE
              /* network admins */
              WHEN scw.primary_on_site_department = 'Executive' THEN 'Regional Admin'
              WHEN scw.primary_on_site_department IN ('Teaching and Learning', 'School Support', 'New Teacher Development') 
               AND scw.primary_job IN ('Achievement Director', 'Chief Academic Officer', 'Chief Of Staff', 'Director', 'Head of Schools'
                                      ,'Director High School Literacy Curriculum', 'Director Literacy Achievement', 'Director Math Achievement'
-                                     ,'Director Middle School Literacy Curriculum') 
+                                     ,'Director Middle School Literacy Curriculum','Head of Schools in Residence','Assistant Dean', 'Assistant School Leader', 'Assistant School Leader, SPED', 'Dean'
```

Comment:
just to clarify: you're making anyone holding these new jobs a sub admin?

Reply:
it's actually just one new job, 'Head of Schools in Residence', but yeah, all these should be sub-admin